Formula/pyrsia.rb

```diff
@@ -12,7 +12,57 @@ def install
     bin.install "pyrsia_node"
   end
 
+  def post_install
+    (var/"pyrsia").mkpath
+  end
+
+  def caveats
+    s = <<~EOS
+      We've installed your Pyrsia Node.
+      To test Pyrsia installation:
+        brew test pyrsia
+      To run Pyrsia Node as a background service:
+        brew services start pyrsia 
+      To check the service status:
+        brew services list
```

Comment:
It might be better to add stop/restart the Pyrsia service

Reply:
I will include the `brew services stop`.

Restart is automatically included by brew. Here is the snippet from the terminal.
```
==> Caveats
We've installed your Pyrsia Node.
To test Pyrsia installation:
  brew test pyrsia
To run Pyrsia Node as a background service:
  brew services start pyrsia 
To check the service status:
  brew services list

To restart dasmanas/pyrsiaanother/pyrsia after an upgrade:
  brew services restart dasmanas/pyrsiaanother/pyrsia
Or, if you don't want/need a background service you can just run:
  RUST_LOG="info,pyrsia=debug" /usr/local/opt/pyrsia/bin/pyrsia_node
```